src/pygama/dsp/processors/invert_polarity.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
+import numpy as np
+from numba import guvectorize
+
+from pygama.dsp.utils import numba_defaults_kwargs as nb_kwargs
+
+
+@guvectorize(
+    ["void(float32[:], float32[:])", "void(float64[:], float64[:])"],
+    "(n)->(n)",
+    **nb_kwargs,
+)
+def invert_polarity(w_in: np.ndarray, w_out: np.ndarray) -> None:
+    """Inverts the polarity of the waveform.
+    Parameters
```

Comment:
```suggestion
    """Inverts the polarity of the waveform.
    
    Parameters
```
The build will fail if you don't have spaces between sections